support-frontend/conf/fastly-snippets/vat-compliance-redirects-recv.vcl

```diff
@@ -1,175 +1,199 @@
 # type: recv
 # fiddle: https://fiddle.fastly.dev/fiddle/6db36311
 if (
-    client.geo.country_code == "KE" ||
```

Comment:
Same as the other one, 24 countries added here and the list was ordered alphabetically